src/Administration/Resources/app/administration/src/core/service/api/document.api.service.js

```diff
@@ -96,7 +96,18 @@ class DocumentApiService extends ApiService {
                     responseType: 'blob',
                     headers: this.getBasicHeaders(),
                 },
-            );
+            )
+            .catch( async (error) => {
```

Comment:
I think you can refer to this part of the line 79 and applying to you code :D 
```suggestion
           .catch((error) => {
                if (error.response?.data?.errors) {
                    this.$listener(
                        this.createDocumentEvent(DocumentEvents.DOCUMENT_FAILED, error.response.data.errors.pop()),
                    );
                }
            });
    }
```

Reply:
As the response type is **blob** we need to turn it into a object. To do this, I used the prototype function ‘text()‘, which returns a promise. So this whole callback needs to async unfortunately

Reply:
Got it, thank you :D